src/modules/getScores.js

```diff
@@ -0,0 +1,14 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const getScores = async (gameId) => {
+  try {
+    const response = await fetch(`${apiUrl}games/${gameId}/scores/`);
+    if (!response.ok) {
+      throw new Error('Failed to fetch game scores');
+    }
+    const { result } = await response.json();
+    return result;
+  } catch (error) {
+    return null;
+  }
+};
+export default getScores;
```

Comment:
[OPTIONAL]
- [ ] Great work implementing the refresh button :clap: However I think it would be better to display the list of games when the page is loaded for better user experience.

Reply:
I will do so, thank you.